packages/jest-matcher-utils/src/index.ts

```diff
@@ -28,12 +28,17 @@ const PLUGINS = [
   AsymmetricMatcher,
 ];
 
+type MatcherHintColor = (arg: string) => string; // subset of Chalk type
```

Comment:
could we use the `Chalk` type directly?

Reply:
Good eyes, good question. A more generic type:

* allows an identity function to keep the default color
* might be needed in future to write application-specific functions based on `ansi-styles` package for snapshot colors, because road map for Chalk 3 implies it might remove some methods; and it will change typings, which might allow cross-version problems?
* however allows caller to rearrange or replace argument text in arbitrary ways :(

Reply:
Yeah, fair enough. Let's keep it they way it is - seems the most future and change proof 🙂